packages/contractkit/src/wrappers/Election.ts

```diff
@@ -0,0 +1,220 @@
+  id: string
+  name: string
+  url: string
+  publicKey: string
+  affiliation: string | null
+}
+
+export interface ValidatorGroup {
+  address: Address
+  id: string
+  name: string
+  url: string
+  members: Address[]
+}
+
+export interface ValidatorGroupVote {
+  address: Address
+  votes: BigNumber
+  eligible: boolean
+}
+
+export interface ElectionConfig {
+  minElectableValidators: BigNumber
+  maxElectableValidators: BigNumber
+  electabilityThreshold: BigNumber
+  maxNumGroupsVotedFor: BigNumber
+}
+
+/**
+ * Contract for voting for validators and managing validator groups.
+ */
+export class ElectionWrapper extends BaseWrapper<Election> {
+  activate = proxySend(this.kit, this.contract.methods.activate)
+  /**
+   * Returns the minimum number of validators that can be elected.
+   * @returns The minimum number of validators that can be elected.
+   */
+  minElectableValidators = proxyCall(
+    this.contract.methods.minElectableValidators,
+    undefined,
+    toBigNumber
+  )
+  /**
+   * Returns the maximum number of validators that can be elected.
+   * @returns The maximum number of validators that can be elected.
+   */
+  maxElectableValidators = proxyCall(
+    this.contract.methods.maxElectableValidators,
+    undefined,
+    toBigNumber
+  )
+  /**
+   * Returns the current election threshold.
+   * @returns Election threshold.
+   */
+  electabilityThreshold = proxyCall(
+    this.contract.methods.getElectabilityThreshold,
+    undefined,
+    toBigNumber
+  )
+  validatorAddressFromCurrentSet = proxyCall(this.contract.methods.validatorAddressFromCurrentSet)
+  numberValidatorsInCurrentSet = proxyCall(
+    this.contract.methods.numberValidatorsInCurrentSet,
+    undefined,
+    toNumber
+  )
+
+  getGroupsVotedFor: (account: Address) => Promise<Address[]> = proxyCall(
+    this.contract.methods.getGroupsVotedFor
+  )
+
+  /**
+   * Returns current configuration parameters.
+   */
+  async getConfig(): Promise<ElectionConfig> {
+    const res = await Promise.all([
+      this.minElectableValidators(),
+      this.maxElectableValidators(),
+      this.electabilityThreshold(),
+      this.contract.methods.maxNumGroupsVotedFor().call(),
+    ])
+    return {
+      minElectableValidators: res[0],
+      maxElectableValidators: res[1],
+      electabilityThreshold: res[2],
+      maxNumGroupsVotedFor: toBigNumber(res[3]),
+    }
+  }
+
+  async getValidatorSetAddresses(): Promise<string[]> {
+    const numberValidators = await this.numberValidatorsInCurrentSet()
+
+    const validatorAddressPromises = []
+
+    for (let i = 0; i < numberValidators; i++) {
+      validatorAddressPromises.push(this.validatorAddressFromCurrentSet(i))
+    }
+
+    return Promise.all(validatorAddressPromises)
+  }
+
+  async getValidatorGroupsVotes(): Promise<ValidatorGroupVote[]> {
+    const validators = await this.kit.contracts.getValidators()
+    const vgAddresses = (await validators.getRegisteredValidatorGroups()).map((g) => g.address)
```

Comment:
How do you feel about expanding to `validatorGroup` here and elsewhere for code clarity vs. the increased verbosity?